app/views/layouts/application.html.erb

```diff
@@ -23,29 +23,30 @@
   <link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond" rel="stylesheet">
 
   <!-- Stylesheets -->
-  <%= stylesheet_link_tag 'application', media: 'all' %>
-  <% if !@extra_css.blank? %>
-    <% @extra_css.each do |css|
-      concat "  #{css}".html_safe
-    end %>
-  <% end %>
+  <%= stylesheet_link_tag 'application' %>
+  <% if @ext_css.present?
+    Array.wrap(@ext_css).each do |css|
+      concat "  #{stylesheet_link_tag css}\n".html_safe
+    end
+  end %>
+  <% if @inline_css.present?
+    Array.wrap(@ext_css).each do |css|
+      concat "  <style>\n#{css}\n  </style>\n".html_safe
```

Comment:
Having looked into `concat` a bit more this morning, I am inclined to say that I still prefer the `<%=` syntax.  Maybe we should do more looking into rails styleguides to see what other people are using, but from surveying about six or so rails apps (blacklight, mastadon, etc) they aren't using `concat` in their views.

The docs also say "The preferred method of outputting text in your views is to use the <%= “text” %> eRuby syntax."  https://apidock.com/rails/ActionView/Helpers/TextHelper/concat

For myself, I think I prefer the `<%=` just because I find it a lot easier to read at a glance, but I'm aware that other languages prefer big blocks like PHP, so I'm sure there are benefits to the above system!  I'm going to leave this PR open for now, merge the orchid one, but I will make an issue to revisit this idea so that we're writing the same kind of code.